project/generate-authors.sh

```diff
@@ -11,5 +11,5 @@ cd "$(dirname "$(readlink -f "$BASH_SOURCE")")/.."
 	# For how it is generated, see `project/generate-authors.sh`.
 	EOH
 	echo
-	git log --format='%aN <%aE>' | sort -uf
+	git log --format='%aN <%aE>' | LC_ALL=C sort -uf
```

Comment:
So this is why we've got a lot of the names moving around here?  Does it work more consistently if we use `C.UTF-8`?